crates/ink/ir/src/ir/item_impl/constructor.rs

```diff
@@ -220,6 +225,22 @@ impl Constructor {
         &self.item.attrs
     }
 
+    /// Returns a list of `cfg` attributes if any.
+    pub fn get_cfg_attrs(&self, span: Span) -> Vec<TokenStream> {
+        self.item
```

Comment:
Maybe we can move this code into some helper function?(we use the same code in the 4 places)